src/modules/commander/Commander.cpp

```diff
@@ -1006,6 +1006,13 @@ Commander::handle_command(const vehicle_command_s &cmd)
 					    && cmd_from_io && (arming_action == vehicle_command_s::ARMING_ACTION_ARM)) {
 						_status.arming_state = vehicle_status_s::ARMING_STATE_IN_AIR_RESTORE;
 					}
+
+				} else {
```

Comment:
This might be easier to understand if it was moved inside the disarm function. 
Here it is an **else** pertaining to an **if** that is related to arm, so not very intuitive

Reply:
Good point! 
Have moved it to the disarm() function.